### DISCUSSION TITLE
Cocoa pods & gradle dependencies?

### DISCUSSION THREAD

**Discussion by PickleCubeSoftwareLtd** [premake/premake-core#2158] [General]
Hi

I am trying to decide to use premake/cmake or Bazel and so far I like the look of premake. 

The main deciding factor for me is how easy it is to include cocoa pods and gradle dependencies in my iOS and Android project? I can't seem to find any example online. 

Thanks

**Comment by Jarod42** [comment]
For simple dependencies, it is mainly just

```
includedirs { "path/to/cocoa/include" }
libdirs {"path/to/cocoa/lib"}
links {"cocoa"}
-- Possibly some defines?
-- defines {"USE_COCOA"}
```
if cocoa is installed as system, `includedirs`/`libdirs` might even (probably) be omitted.

(But it would also be relatively simple in any build system).

If you have to rebuild cocoa, using same buildsystem than cocoa would reduce learning stage.
